### DISCUSSION TITLE
Unknown Issue

### DISCUSSION THREAD

**Discussion by riku-02** [jonathanong/heroku-buildpack-ffmpeg-latest#43] [Q&A]
I didn't got any error with this buildpack, my discord music bot is playing but the sound doesn't come out, it only works with my local ffmpeg installed on my computer.

**Comment by tthn0** [comment]
Me too

**Comment by JuJu2181** [comment]
I tried to use it in my discord music bot, My bot worked well with local ffmpeg in my device but when I tried it in heroku with this buildpack it give me error saying too many http requests. Any Solutions please

**Comment by tthn0** [comment]
> I tried to use it in my discord music bot, My bot worked well with local ffmpeg in my device but when I tried it in heroku with this buildpack it give me error saying too many http requests. Any Solutions please

Nope I haven't figured out how to get it working on Heroku
